tests/unittests/conftest.py

```diff
@@ -376,14 +394,56 @@ def fake_issue_with_labels_json(monkeypatch, fake_issue_json):
 @dataclass
 class MockJiraApiResponse:
     _json: dict | list = None
-    status_code: int = 200
+    _status_code: int = None
+    _ok: bool = None
+    _content: bytes = None
+    _headers: dict = None
+    _url: str = None
 
+    @property
     def ok(self):
-        return True
+        if not self._ok:
+            if str(self.status_code).startswith("2"):
+                self._ok = True
+            else:
+                self._ok = False
+        return self._ok
+
+    @property
+    def status_code(self):
+        if not self._status_code:
+            self._status_code = 200
+        return self._status_code
 
     def json(self):
+        if not self._json:
```

Comment:
ditto, for all

Reply:
See [previous comment](https://github.com/RedHatQE/firewatch/pull/175/files#r1539482187)